src/services/user.service.ts

```diff
@@ -31,3 +34,13 @@ export async function createUser(user: User): Promise<User> {
     },
   })) as User;
 }
+
+export async function isPasswordValid(user: User, password: string) {
+  return await bcrypt.compare(user.password, password);
+}
+
+export function generateAccessToken(data: { id: number; role: string }) {
```

Comment:
Also here please add return value.

Reply:
If you're referencing `generateAccessToken`, it already has a return statement. It returns the result of calling `jwt.sign()`, which, as I understand, creates a JWT token. Is that right?

Reply:
should be `generateAccessToken(data: { id: number; role: string }): string`

Reply:
Fixed.